wsclient/src/test/java/org/hpccsystems/ws/client/BaseRemoteTest.java

```diff
@@ -240,7 +279,16 @@ public boolean verify(String hostname,javax.net.ssl.SSLSession sslSession)
         // Run the generate-datasets.ecl script if present in the project resources
         try
         {
-            executeECLScript("generate-datasets.ecl");
+             //Span eclscriptspan = tracer.spanBuilder("generate-datasets.ecl").startSpan();
```

Comment:
I think it could be interesting to add tracing into our tests. This could be a good way to better understanding how the tracing works in practice, and it could help us track down issues. So, I think lets keep this trace code